doc/docgen.rst

```diff
@@ -153,7 +153,7 @@ Output::
     ]
   }
 
-Similarly to the old ``doc`` command the old ``jsondoc`` command has been
+Similarly to the old ``doc`` command, the old ``jsondoc`` command has been
 renamed ``jsondoc0``.
```

Comment:
Maybe change it to "renamed to" here?

Reply:
You're correct. Fixed :)

(There were literally many many littles issues - I tried to tackle them all. I hope I did a good job...)